### ISSUE TITLE
workbench.action.terminal.paste pastes into editor not terminal after update to 1.26

### ISSUE THREAD

**Issue by clayms** [Microsoft/vscode#57271]
With the settings below, "workbench.action.terminal.paste" duplicates the line the cursor is currently on in the editor, rather than pasting it into the active terminal like it used to before updating to 1.26.1 . 

Commenting out the  `"workbench.action.terminal.paste"` line in the user settings file, results in the cursor just moving down in the editor as expected.  

Did the `"workbench.action.terminal.paste"` command change?

- VSCode Version: 1.26.1
- OS Version: Win 10 Enterprise 64bit
- https://github.com/geddski/macros 1.0.0

Steps to Reproduce:

user settings file:

```
{
    "macros": {
        "runLine": [
            "editor.action.clipboardCopyAction",
            "workbench.action.terminal.paste",
            "cursorDown",
            {"command": "workbench.action.focusActiveEditorGroup", "when": "terminalFocus"}
        ]
    }
}
```

keybindings.json 

```
[
    {
        "key": "ctrl+enter",
        "command": "macros.runLine",
        "args": {"cmd": "ls", "match": ".*"},
        "when": "editorTextFocus"
    }
]
```

**Comment by clayms** [comment]
Problem persists when all extensions except for `macros` are disabled.  

Also, the `{"command": "workbench.action.focusActiveEditorGroup", "when": "terminalFocus"}` line is not working as it used to before upgrading to version 1.26.  When the line the cursor is on in the editor is duplicated in the editor, then the terminal gets the focus and the focus never comes back to the editor like it used to.

**Comment by clayms** [comment]
Is this still the correct file?
https://github.com/Microsoft/vscode/blob/f82b14800cc0ed6f21a450b684518a4db05fc260/src/vs/workbench/parts/terminal/common/terminalCommands.ts#L32

Most of the other commands from this file work as expected when put in my macro.  What is the problem with `"workbench.action.terminal.paste"` ?

**Comment by clayms** [comment]
downgraded to 1.25 and everything works just fine again.  Please let me know when you fix this issue.

**Comment by Tyriar** [comment]
@clayms this will be fixed in 1.27.

**Comment by Tyriar** [comment]
To verifier: please also make sure the terminal is focused under these circumstances:

- Reload window without terminal visible, show terminal
- Reload window with output panel visible, click terminal panel
- Reload window with output panel visible, run toggle terminal command (ctrl+\`)
